lib/gooddata/models/blueprint/dataset_blueprint.rb

```diff
@@ -144,7 +144,7 @@ def self.labels(dataset)
         find_columns_by_type(dataset, :label)
       end
 
-      def self.reference_label_for_attribtue(dataset, attribute)
+      def self.reference_label_for_attribute(dataset, attribute)
```

Comment:
As all the SDK methods are in essence part of an API, we should provide backwards compatibility - there is either an `alias_method` functionality in ruby (IMO an uglier solution as the method definition would still have a typo) or we can simply define another method with the original name that just calls the new method and passes it all the params.

Reply:
Not sure if I got that properly: please see the fixup commit b272fbb76b70592c69e620c0f6941988979244da

Reply:
@kabanoid looks good; an even better approach would be with the splat operator to pass all params without explicitly mentioning them - https://alexcastano.com/everything-about-ruby-splats/#the-simple-splat . I'd also feel better with a comment that explains why we decided to have this method with a typo.